examples/config-anonymous-authz.json

```diff
@@ -8,29 +8,31 @@
     "port": "8080",
     "realm": "zot",
     "accessControl": {
-      "**": {
-        "anonymousPolicy": [
-          "read",
-          "create"
-        ]
-      },
-      "tmp/**": {
-        "anonymousPolicy": [
-          "read",
-          "create",
-          "update"
-        ]
-      },
-      "infra/**": {
-        "anonymousPolicy": [
-          "read"
-        ]
-      },
-      "repos2/repo": {
-        "anonymousPolicy": [
-          "read"
-        ]
-      }
+      "repos": {
```

Comment:
This is a breaking change actually!

Reply:
In the commit description/message, follow this approach

BREAKING CHANGE: a commit that has the text BREAKING CHANGE: at the beginning of its optional body or footer section introduces a breaking API change (correlating with [MAJOR](http://semver.org/#summary) in semantic versioning). A breaking change can be part of commits of any type. e.g., a fix:, feat: & chore: types would all be valid, in addition to any other type.

Reply:
feat(groups): ...
<NEWLINE>
BREAKING CHANGE: <only related to breaking change>
<NEWLINE>
Explanation of the commit itself.

Reply:
I changed the commit message